Vagrantfile_RPM_Test

```diff
@@ -0,0 +1,90 @@
+# -*- mode: ruby -*-
+# # vi: set ft=ruby :
+
+# Require Vagrant 2.0+.
+Vagrant.require_version '>= 2.0.0'
+
+# Setting up globals from YAML configuration file.
+settings = YAML::load_file('config.yml')
+$pg_yum_repo = settings.fetch('yum-repos')['hoot-pgdb95-repo']
+$hoot_yum_repo = settings.fetch('yum-repos')['hoot-develop-repo']
+
+Vagrant.configure("2") do |config|
+    tomcatPort = ENV['TOMCAT_PORT']
+    if tomcatPort.nil?
+        tomcatPort = '8888'
+    end
+
+    transPort = ENV['NODEJS_PORT']
+    if transPort.nil?
+        transPort = '8094'
+    end
+
+    mergePort = ENV['P2P_PORT']
+    if mergePort.nil?
+        mergePort = '8096'
+    end
+
+    disableForwarding = ENV['DISABLE_VAGRANT_FORWARDING']
+    if disableForwarding.nil?
+        config.vm.network "forwarded_port", guest: 8080, host: tomcatPort
+        config.vm.network "forwarded_port", guest: 8094, host: transPort
+        config.vm.network "forwarded_port", guest: 8096, host: mergePort
+    end
+    
+    config.vm.provider "virtualbox" do |vb|
+        vb.gui = false
+        vb.memory = 10240
+        vb.cpus = 4
+    end
+    
+    def aws_provider(config, os)
+        config.vm.provider :aws do |aws, override|
+            override.nfs.functional = false
+            aws.instance_type = ENV.fetch('AWS_INSTANCE_TYPE', 't2.xlarge')
```

Comment:
Any particular reason you chose the 't' aws instance type?

Reply:
no particular reason, I just reviewed the cost and specs of the instance and thought that'd be a reasonable machine. Would you recommend a different type?

Reply:
No.  If it's working, I'd say leave as is.